be2-scala/src/test/scala/ch/epfl/pop/pubsub/graph/handlers/WitnessHandlerTest.scala

```diff
@@ -0,0 +1,125 @@
+package ch.epfl.pop.pubsub.graph.handlers
+
+import akka.actor.{Actor, ActorSystem, Props, Status}
+import akka.pattern.AskableActorRef
+import akka.testkit.{ImplicitSender, TestKit}
+import akka.util.Timeout
+import ch.epfl.pop.model.network.method.message.Message
+import ch.epfl.pop.model.objects.{Base64Data, DbActorNAckException, Hash, Signature, WitnessSignaturePair}
+import ch.epfl.pop.pubsub.graph.PipelineError
+import ch.epfl.pop.storage.DbActor
+import ch.epfl.pop.storage.DbActor.DbActorAddWitnessMessage
+import org.scalatest.{BeforeAndAfterAll, FunSuiteLike, Matchers}
+import util.examples.data.WitnessMessages
+import util.examples.lao.CreateLaoExamples.{SENDER, createLao}
+
+import scala.concurrent.duration.FiniteDuration
+
+
+class WitnessHandlerTest extends TestKit(ActorSystem("Witness-DB-System")) with FunSuiteLike with ImplicitSender with Matchers with BeforeAndAfterAll {
+  // Implicits for system actors
+  implicit val duration: FiniteDuration = FiniteDuration(5, "seconds")
+  implicit val timeout: Timeout = Timeout(duration)
+
+  override def afterAll(): Unit = {
+    // Stops the testKit
+    TestKit.shutdownActorSystem(system)
+  }
+
+  //arrange
```

Comment:
unless you keep them around to fully stick to the aaa pattern, I would remove the "useless" comments relating to it

Reply:
(very minor detail)